inst/scripts/insert_image.R

```diff
@@ -20,8 +20,13 @@ local({
         height = '90px'
       ),
       shiny::fillRow(
-        txt_input('w', 'Width', '', '(optional) e.g., 400px or 80%'),
-        txt_input('h', 'Height', '', '(optional) e.g., 200px'),
+        txt_input('w', 'Width', '', '(optional) e.g., 400px'),
+        txt_input('h', 'Height', '', '(optional) e.g., 80%'),
+        shiny::column(width = 1, offset = 1, shiny::radioButtons(
+          'usebaseurl', 'Use base url in links?',
+          inline = TRUE, c('Yes' = TRUE, 'No' = FALSE),
+          selected = getOption('blogdown.insertimage.usebaseurl', FALSE)
+        )),
```

Comment:
I actually meant to get rid of the radio buttons here, and use `getOption('blogdown.insertimage.usebaseurl', FALSE)` instead of `input$usebaseurl` below. I believe users will want to set this option globally instead of per-image basis, so basically this option will go to `.Rprofile`, and they don't need to see the option in the addin.